academy/4-my-own-chain/game-fifo.md

```diff
@@ -183,9 +183,9 @@ Now that the new fields are created, you need to update them accordingly to keep
     }
     ```
 
-    Same remark here about using `SetStoredGame` and `SetNextGame` after calling this function.
+    Again, it is advisable to do `SetStoredGame` and `SetNextGame` after calling this function.
 
-## Use it
+## Implementation
```

Comment:
In software development, _implementation_ usually has the meaning of putting content where a void was left on purpose to be _implemented_:

```typescript
function getValue(key: string): ValueType {
    throw new Error("TODO implement");
}
```

As opposed to what is being done here, which is take newly built functions, and call them within existing code. This existing code is not empty and works well already; it is being augmented by the new line.

I am sure we can do better than _Use it_, though.

Reply:
"Integration"? Or an equivalent phrase: "Integrating FIFO"

Reply:
> "Integration"? Or an equivalent phrase: "Integrating FIFO"

Yes